docs/user-documentation/glossary.md

```diff
@@ -0,0 +1,79 @@
+The following glossary of terms addresses an Islandora 8 context. When comparing new Islandora and Fedora to older versions it may also be helpful to reference [the Islandora 7 Glossary](https://wiki.duraspace.org/display/ISLANDORA/APPENDIX+E+-+Glossary).
+ 
+## CLAW
+ CLAW (CLAW Linked Asset WebDataFrameWork) was the production code name for the software released in June, 2019 as _Islandora 8_
```

Comment:
Wouldn't it be 'development code name'? Also, we need a period at the end.

Reply:
🤦‍♀ yes, _development_